about.html

```diff
@@ -0,0 +1,139 @@
+            <h1>THE CREATIVE GLOBAL SUMMIT 2021</h1>
+          </div>
+          <div class="about-paragraph">
+            <div class="summary-header">
+              <span>Summit Summary</span>
+              <hr>
+            </div>
+            <p>Mission is to promote and unite the integration of refugees, immigrants and the community at large into a
+              self-sufficient, healthy and inclusive multi-ethnic society.
+              By the mid-'80s, ICSC joined forces with another community-based organization, Southeast Asian Refugee
+              Federation (SEARF).
+              followed by the establishment of Asia House in 2006.
+            </p>
+            <span>
+              <span class="sub-cont">
+                <!-- <img src="Images/about_icon.png" alt="about-ico"> -->
+                <p>2022 December Global Refugee Summit</p>
+                <p id="ke">United State</p>
+              </span>
+            </span>
+          </div>
+          <div class="reach-us-out">
+            <span>Please contact us as on email on Mile Phone Number for more info about the UN-Globe
+              2022 Refugees Summit.
+            </span>
+            <span id="mail">@unglobal-refugees.org</span>
+          </div>
+        </div>
+        <div class="rest">
+          <div class="un-logo">
+            <span>
+              <img src="Images/logo.png" alt="">
+            </span>
+            <span>THE UN-REFUGEE <span class="sp-2">summit</span></span>
+          </div>
+          <div class="gallery">
+            <div class="un-activities">
+              <span>Our Activities</span>
+              <hr>
+            </div>
+            <span id="p">The are the two ocaasions where the globals summit
+              came to agreement and constructed more camps world wide</span>
+            <div class="pics">
+              <div class="img-cont">
+                <img src="Images/ref_1.png" alt="gallery-1">
+                <div class="overlay1">
+                  <div class="text">
+                    <span>3st June 2022</span>
+                    <p>America National Conference Centre</p>
+                  </div>
+                </div>
+              </div>
+              <div class="img-cont">
+                <img src="Images/ref_2.jpg" alt="">
+                <div class="overlay1">
+                  <div class="text">
+                    <span>1st September 2022</span>
+                    <p>America National Conference Centre</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <footer id="footnote">
+          <div class="footer">
+            <span><img src="Images/cc-logo2.png" alt=""></span>
+            <span class="txt">&copy;2022 UN Human Rights and Refugee Affairs</span>
+          </div>
+        </footer>
+      </section>
+
+   
+    <script src="nav.js"></script>
+
+</body>
+
+</html>
```

Comment:
- [ ] [OPTIONAL] You make a great effort to build this project but we can see some differences with the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), I would like to recommend you work on the design again. As a developer, it is very important to pay attention to details and be able to deliver good work :+1: